public/react/components/Form.jsx

```diff
@@ -1,33 +1,97 @@
 import React, { useEffect, useState } from 'react'
 import apiURL from '../api'
 
-export const Form = ({ setFormIsActive }) => {
+export const Form = ({ setFormIsActive, selectedItem, fetchItems }) => {
+  const [name, setName] = useState(selectedItem?.name || '')
+  const [description, setDescription] = useState(selectedItem?.description || '')
+  const [category, setCategory] = useState(selectedItem?.category || '')
+  const [price, setPrice] = useState(selectedItem?.price || '')
+  const [imgUrl, setImgUrl] = useState(selectedItem?.imgUrl || '')
+
+  // ADD/UPDATE the item
+  async function onHandleClick (e) {
+    e.preventDefault()
+    const itemData = {
+      name,
+      description,
+      category,
+      price,
+      imgUrl
```

Comment:
```suggestion
      image
```
this wouldn't add/update the item correctly as the field should be `image` to match the model/router etc... 

I am also not sure if the order is important as the data currently generates in the order `name`, `price`, `description`, `category`, `image` and this is the order set in the model which is slightly different to what you have here